### DISCUSSION TITLE
Previous version of tailwind elements

### DISCUSSION THREAD

**Discussion by EliaGut** [mdbootstrap/TW-Elements#1711] [Support from community] [answered]
Hi all,

Im using in my project older previous version 2 of CSS/JS Tailwind CND links. Where can I find HTML elements e.g. for Accordion/Modals so it works with my older css/js tailwind version, because new version elements seen on website are not working =(

Thanks much, EliaGut

**Comment by codedbyfrancis** [comment]
Hello,
I have the same issue. Did you find it? I need the [v1.0.0-alpha ](https://cdn.jsdelivr.net/npm/tw-elements/dist/js/index.min.js)

**Reply by EliaGut** [reply]
Yes, i found some dirty way to fine old HTMLs in **web.archive** https://web.archive.org/

**Reply by codedbyfrancis** [reply]
Thanks a lot, I got some improvements, the modal now works but the accordion does not. I mean whenever I click on it, the contents show for a quick second and disappear. What did you do with CSS? What did you do in your issue?

**Comment by juujisai** [comment, marked as answer]
Hi @EliaGut @codedbyfrancis 
You can find the alpha13 cdn here:

- https://cdn.jsdelivr.net/npm/tw-elements@1.0.0-alpha13/dist/js/index.min.js
- https://cdn.jsdelivr.net/npm/tw-elements@1.0.0-alpha13/dist/css/index.min.css
